.travis.yml

```diff
@@ -9,8 +9,18 @@ stages:
   - test
   - browser-test
 
+before_install:
+  - npm install -g npm@6
+  - npm install -g greenkeeper-lockfile@1
```

Comment:
@josdejong In these two lines I install npm and greenkeeper-lock file. Currently, we install the latest minor version. Should we be more specific?

Reply:
I like it like this. 

If we want to be more specific some day we can always change it. But then we need some greenkeeper solution to keep the `.travis.yml` file up to date too ;)

Reply:
ping @harrysarson

Reply:
Greenkeeper-ception :laugh: